app/routes/agenda/agendaitems.js

```diff
@@ -117,6 +117,24 @@ export default class AgendaAgendaitemsRoute extends Route {
     controller.scrollToAnchor();
   }
 
+  @action
+  loading(transition) {
+    // eslint-disable-next-line ember/no-controller-access-in-routes
+    const controller = this.controllerFor(this.routeName);
+    controller.isLoadingModel = true;
+    transition.promise.finally(() => {
+      controller.isLoadingModel = false;
+    });
+
+   // only bubble loading event when transitioning between tabs
+   // to enable loading template to be shown
```

Comment:
Is this comment up-to-date?  'tabs' are not applicable here? 
I assume we want to say "don't bubble loading event if only query params change"?

Reply:
I just copy-pasted this snippet to be honest, but I think it still applies? We still want to show the template-level loading state when switching between the Overzicht, Detail, Vergelijken, and Documenten tab, right?

But the comment is somewhat confusing and really just an artifact of being copied around, I will try to make it a bit clearer